packages/mobile/src/components/lineup-tile/TrackTile.tsx

```diff
@@ -93,6 +93,10 @@ const TrackTileComponent = ({
     track_id
   } = track
 
+  const currentScreen = navigation.getState().history?.[0]
+  // @ts-expect-error
```

Comment:
how come?

Reply:
weirdly, history returns undefined[]... what do you think?